Source/ChilliSource/Core/Base/Application.h

```diff
@@ -430,6 +443,7 @@ namespace ChilliSource
         ///
         void Destroy() noexcept;
         
+        std::vector<RenderSnapshot> m_currentRenderSnapshots;
```

Comment:
Minor: 'm_pendingRenderSnapshots' may be a better name? Conveys that it's waiting for something rather than persistent.